maps_dashboards/public/model/mapLayerType.ts

```diff
@@ -36,6 +36,8 @@ export type DocumentLayerSpecification = {
     geoFieldType: 'geo_point' | 'geo_shape';
     geoFieldName: string;
     documentRequestNumber: number;
+    showTooltips: boolean;
```

Comment:
We might also need to update the initial config here: 
https://github.com/opensearch-project/dashboards-maps/blob/feature/new-maps/maps_dashboards/public/utils/getIntialLayerConfig.ts

Reply:
updated with default values